int16 command
---
string state
